src/django_dramatiq_pg/migrations/0001_initial.py

```diff
@@ -21,20 +20,20 @@ class Migration(migrations.Migration):
                     "state",
                     models.TextField(
                         choices=[
-                            ("QUEUED", "queued"),
-                            ("CONSUMED", "consumed"),
-                            ("REJECTED", "rejected"),
-                            ("DONE", "done"),
+                            ('queued', 'Queued'),
+                            ('consumed', 'Consumed'),
+                            ('rejected', 'Rejected'),
+                            ('done', 'Done'),
```

Comment:
i think the previous implementation might've been a mistake; the LHS of those tuples shouldn't've been upper..
your change looks good now.